code/midtown/midtown.cpp

```diff
@@ -317,6 +317,12 @@ static void MainPhase(i32 argc, char** argv)
         {
             no_audio = true;
         }
+        else if (ARG("-cop2opp"))
+        {
+            CHEATING = true;
+
+            MMSTATE.ChaseOpponents = true;
+        }        
```

Comment:
It would be better if this was moved into `mmStatePack::ParseStateArgs`

Reply:
You're right, I found it in "state.cpp". 

Wanted to update the commit earlier, but need to learn Github better.
So if you don't mind, keep the PR open for now.